.ci_support/linux_.yaml

```diff
@@ -9,4 +9,4 @@ pin_run_as_build:
     min_pin: x.x
     max_pin: x.x
 python:
-- '2.7'
+- 3.6.* *_cpython
```

Comment:
OmegaConf 2.0 supports PYthon >= 3.6.
This does not seem to correctly represent that.

Reply:
This file is autogenerated and should not be modified. I'll take care of the python requirements elsewhere.